core/src/main/java/com/linecorp/armeria/client/logging/ContentPreviewingClient.java

```diff
@@ -159,6 +159,6 @@ public HttpResponse execute(ClientRequestContext ctx, HttpRequest req) throws Ex
 
         ctx.logBuilder().defer(RequestLogProperty.RESPONSE_CONTENT_PREVIEW);
         final HttpResponse res = unwrap().execute(ctx, req);
-        return setUpResponseContentPreviewer(contentPreviewerFactory, ctx, res, responsePreviewSanitizer);
+        return ResponseContentPreviewer.of(contentPreviewerFactory, ctx, responsePreviewSanitizer).setUp(res);
```

Comment:
Should we do this only when `res.isEmpty()` is `false`, like we did so for `req`?